### PR TITLE
Fork the bookmarks manager hotkey

### PR DESCRIPTION
**Pull request by twisterghost**
This is a potential fix for #926. Use Shift+Alt+B to open the bookmarks manager on windows since Ctrl+Alt+B seems to be often hijacked.

As far as I can tell the only other way to fix this to make it work universally is to add a hotkey for `ctrl+alt+b` to `localShortcuts.js` to open a new window pointing at `about:bookmarks`. I have a working copy of that if that is preferred.

### CODE REVIEW

File: js/commonMenu.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ module.exports.preferencesMenuItem = {
 
 module.exports.bookmarksMenuItem = {
   label: 'Bookmarks manager...',
-  accelerator: 'CmdOrCtrl+Alt+b',
+  accelerator: process.platform === 'win32' ? 'Shift+Alt+B' : 'CmdOrCtrl+Alt+B',
```
Comment (bbondy) [inline js/commonMenu.js]: Right idea to fork it but pls use the same shortcut as Chrome here for familiarity's sake.  That's  Control+Shift+O for bookmarks manager on Windows.  And Command+Alt+B for OS X.

Comment (twisterghost) [inline js/commonMenu.js]: Will do

Comment (bbondy) [inline js/commonMenu.js]: Actually Linux is always the same as Windows as well so this should be `const isDarwin = process.platform === 'darwin'` w/ an isDarwin ? X : Y.

Comment (bbondy) [inline js/commonMenu.js]: See `app/menu.js:` we usually put that const at the top

Comment (twisterghost) [inline js/commonMenu.js]: But wouldnt we want it to be windows or darwin?

Comment (bbondy) [inline js/commonMenu.js]: we want windows + linux to be Command+Shift+O and OS X to be 'CmdOrCtrl+Alt+B',

Comment (twisterghost) [inline js/commonMenu.js]: 024ebdc

**Comment by bbondy** [comment]
++ great fixes, thanks!

**Pull request merged**